analyzer-models/src/test/java/ffb/analyzer/models/espn/TeamRecordOverallTests.java

```diff
@@ -28,7 +23,7 @@ public static void buildPrep() {
     }
 
     @Test
-    public void testTeamSerialization() throws JsonProcessingException, MalformedURLException {
+    public void testTeamSerialization() throws JsonProcessingException {
```

Comment:
There's a `testSerialization` method you should override instead of `testTeamSerialization`

Reply:
Updated this to override testSerialization instead of being a separate test. I did the same TransactionCounterTests, TeamRecordTests, and TeamRecordOverallTests